cluster-service/deploy/templates/cloud-resources-config.configmap.yaml

```diff
@@ -9,6 +9,24 @@ data:
   instance-types.yaml: |
     instance_types:
     # DEMO INSTANCE TYPES
+    - id: Standard_D2s_v3
+      name: Standard_D2s_v3 - General purpose
+      cloud_provider_id: azure
+      cpu_cores: 2
+      memory: 8589934592
+      category: general_purpose
+      size: d2s_v3
+      ccs_only: true
```

Comment:
Is that attribute needed, as it's not added to the other type?

IIRC it stands for **customer cloud subscription** for validation in CS. I'm unsure if that applies to ARO HCP, as all ARO HCP clusters will be provisioned into the customers' accounts.